docs/source/terms_and_model.rst

```diff
@@ -342,11 +342,122 @@ Location for Variation.
 **Implementation Guidance**
 
 * Location refers to a position.  Although it MAY imply a sequence,
-  the two concepts are not interchangable, especially when the
+  the two concepts are not interchangeable, especially when the
   location is non-specific (e.g., a range) or symbolic (a gene).
 
 
+.. _chromosome-location:
+
+ChromosomeLocation
+$$$$$$$$$$$$$$$$$$
+
+**Biological definition**
+
+Imprecise chromosomal locations based on named landmarks assayed
+through chromosomal staining, FISH, recombination studies, or other
+low-resolution methods.
+
+**Computational definition**
+
+A ChromosomeLocation is a :ref:`Location` that is defined by a named
+chromosomal band within a species.
+
+**Information model**
+
+.. list-table::
+   :class: reece-wrap
+   :header-rows: 1
+   :align: left
+   :widths: auto
+
+   * - Field
+     - Type
+     - Limits
+     - Description
+   * - _id
+     - :ref:`CURIE`
+     - 0..1
+     - Location Id; MUST be unique within document
+   * - type
+     - string
+     - 1..1
+     - Location type; MUST be set to **'ChromosomeLocation'**
+   * - species
+     - :ref:`CURIE`
+     - 1..1
+     - An external reference to a species taxonomy.  See Implementation Guidance, below.
+   * - chr
+     - string
+     - 1..1
+     - The symbolic chromosome name
+   * - start
+     - string
+     - 1..1
+     - The start chromosomal band name
+   * - end
+     - string
+     - 1..1
+     - The end chromosomal band name
+
+
+**Implementation guidance**
+
+* ChromosomeLocation is intended to enable the representation of
+  cytogenetic results from karyotyping or low-resolution molecular
+  methods.  Precise :ref:`SequenceLocation` should be preferred when
+  nucleotide-scale location is known.
+* `species` is specified using the NCBI taxonomy.  The CURIE prefix
+  MUST be `taxonomy`, corresponding to the `NCBI taxonomy prefix at
+  identifiers.org
+  <https://registry.identifiers.org/registry/taxonomy>`__.
```

Comment:
We should also specify the expected content for the reference portion of the CURIE.

```suggestion
  <https://registry.identifiers.org/registry/taxonomy>`__, and the CURIE reference MUST be an NCBI taxonomy identifier (e.g., 9606 for Homo sapiens).
```